mac

```diff
@@ -150,6 +150,11 @@ brew_install_or_upgrade 'node'
 brew_install_or_upgrade 'rbenv'
 brew_install_or_upgrade 'ruby-build'
 
+brew_install_or_upgrade 'caskroom/cask/virtualbox'
```

Comment:
Using cask, it'll install the virtual box dependency for you if you do:

~~~
brew_tap 'caskroom/cask'
brew_install_or_upgrade 'brew-cask'
brew cask install dockertoolbox
~~~